src/components/CreateReminders/CreateReminder.js

```diff
@@ -75,19 +75,16 @@ const CreateReminders = () => {
     const reminderDueDate = dateInputRef.current.value;
     const reminderName = nameInputRef.current.value;
     const reminderNotes = notesRef.current.value;
-
-    const dueDate = moment(reminderDueDate).format("YYYY MMMM dddd")
-
-    let calendarNow = moment().calendar()
-    console.log(calendarNow)
+    const dueDate = moment(reminderDueDate).format("MMM Do YY")
```

Comment:
@sebastianserrano  updated and removed console.logs

Reply:
Sounds good